run_backup.py

```diff
@@ -197,22 +181,18 @@ def main():
                 datasets.append(dataset)
     except Exception as ex:
         print "Failed to retrieve local datasets"
-        if verbosity:
-            print ex
+        debug(1, ex)
         sys.exit(1)
 
     pool = datasets[0]
 
     for dataset in datasets:
-        if verbosity > 1:
-            print "Processing dataset " + dataset
+        debug(2, "Processing dataset " + dataset)
         if args.initbackup:
-            if verbosity:
-                print "initbackup"
+            debug(1, "initbackup")
             prevsnap = ''  # We don't have a snapshot to refer to for initial backup
             for snap in getsnapshots(dataset, 0):
-                if verbosity > 1:
-                    print "Sending snapshot " + snap
+                debug(2,"Sending snapshot " + snap)
                     if not sendsnapshot(prevsnap, snap, args.ip, args.mem, args.port, args.backuppool, pool):
```

Comment:
Wrong indentation, was already wrong in source branch, but now creates an indentation error